### PR TITLE
Implement delete keywords sets

### PR DESCRIPTION
**Pull request by juanuribe28**
The user is now capable of erasing any keyword set that appears in the previous keywords modal window.

### CODE REVIEW

**Review by dranilew — APPROVED** [review]

File: src/main/java/com/google/sps/servlets/DeleteKeywordsServlet.java [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.KeyFactory;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+/** Servlet that deletes the keywords.*/
```
Comment (daperlmut) [inline src/main/java/com/google/sps/servlets/DeleteKeywordsServlet.java]: [Nit] Add a space before '*/'

File: src/test/java/com/google/sps/tests/DeleteKeywordsServletTest.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+    DELTE_KEYWORDS_SERVLET.doPost(mockedRequest, mockedResponse);
+    Mockito.verify(mockedRequest, Mockito.atLeast(1)).getParameter("k");
+  }
+
+  @Test
+  public void testServlet() throws IOException {
+    DatastoreService mockedDatastoreService = PowerMockito.mock(DatastoreService.class);
+    setUpDatastoreMock(mockedDatastoreService);
+
+    String expected = "ahpzfnN0ZXAyMDIwLXRlYW0zNi1jYXBzdG9uZXIUCxIHS2V5d29yZBiAgICYtMiZCgw";
```
Comment (daperlmut) [inline src/test/java/com/google/sps/tests/DeleteKeywordsServletTest.java:49]: Is this just a random string?  I'm confused as to why we have such a long and complicated value here.

**Review by daperlmut — CHANGES_REQUESTED** [review]

File: src/test/java/com/google/sps/tests/DeleteKeywordsServletTest.java (hunk 2 above)
Comment (juanuribe28) [reply src/test/java/com/google/sps/tests/DeleteKeywordsServletTest.java:49]: It is a real key string. They have to encode the entity and the id.

Comment (daperlmut) [reply src/test/java/com/google/sps/tests/DeleteKeywordsServletTest.java:49]: Hmm, it looks a bit odd.  Maybe at the least we should add a comment to indicate what this is an encoding of.

**Review by daperlmut — CHANGES_REQUESTED** [review]

**Review by daperlmut — APPROVED** [review]

**Pull request merged**